### DISCUSSION TITLE
IPVM WG Vancouver Feb 2023

### DISCUSSION THREAD

**Discussion by bmann** [ipvm-wg/spec#14] [Workshop]
Thread for notes, discussion, and other links for the Feb 2023 in person WG event. 

Discord backchannel is in [Fission Discord](https://fission.codes/discord) in the [#events thread](https://discord.com/channels/478735028319158273/1077414705938845716).

**Comment by expede** [comment]
I've also created a Workshop category in lieu of nested threading
